[FIX] dependency on 'dateutil' does not appear to be valid

ozoromo opened the pull request:
Fixes:
`odoo odoo.addons.base.models.ir_module: python external dependency on 'dateutil' does not appear to be a valid PyPI package. Using a PyPI package name is recommended.`

ozoromo committed:
Update __manifest__.py

Fixes:
`odoo odoo.addons.base.models.ir_module: python external dependency on 'dateutil' does not appear to be a valid PyPI package. Using a PyPI package name is recommended.`

pedrobaeza commented:
Please change the PR title and commit message to one complying with the OCA expected format: 

https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#71commit-message

About the content itself, as `dateutil` is part of Odoo requirements, we can simply remove the line:

https://github.com/odoo/odoo/blob/15.0/requirements.txt#L34

ozoromo committed:
Removed redundant dependency declaration in __manifest__.py

Removed redundant dependency declaration. (dateutils is already a Odoo dependency)

ozoromo commented:
@pedrobaeza It seems simply removing the line causes errors

ozoromo committed:
Undo last commit

Undo commit:
#7a1f1d08f65734362499ed07d24c73407163587a

pedrobaeza commented:
Which error?

The commit message is still incorrect.

ozoromo commented:
> Which error?
https://github.com/OCA/contract/actions/runs/5333205012/jobs/9663374705

I'll close this PR now and reopen it in a second from another branch to avoid changes to Nitrokey:15.0 being added to this PR
